I0516 11:49:50.774511 140613858928448 run.py:28] Device: cuda
W0516 11:49:51.180024 140613858928448 utils.py:474] Not applying transforms to coordinates in unrecognized space 'ale_2mm'
I0516 11:49:51.262671 140613858928448 diagnostics.py:438] 5/159 coordinates fall outside of the mask. Removing them.
/well/nichols/users/pra123/anaconda3/envs/torch/lib/python3.11/site-packages/patsy/util.py:672: FutureWarning: is_categorical_dtype is deprecated and will be removed in a future version. Use isinstance(dtype, CategoricalDtype) instead
  return _pandas_is_categorical_dtype(dt)
/well/nichols/users/pra123/anaconda3/envs/torch/lib/python3.11/site-packages/patsy/util.py:672: FutureWarning: is_categorical_dtype is deprecated and will be removed in a future version. Use isinstance(dtype, CategoricalDtype) instead
  return _pandas_is_categorical_dtype(dt)
slurmstepd: error: *** JOB 58558335 ON compg036 CANCELLED AT 2024-05-16T15:49:06 DUE TO TIME LIMIT ***
slurmstepd: error: *** STEP 58558335.0 ON compg036 CANCELLED AT 2024-05-16T15:49:06 DUE TO TIME LIMIT ***
